### DISCUSSION TITLE
Current directory only, non subdirectories?

### DISCUSSION THREAD

**Discussion by HubKing** [rhash/RHash#205] [Q&A] [answered]
`rhash -r .` works but it processes subdirectories. `rhash .` just prints "RHash: .: Is a directory". What if I want to hash files in a directory but not its subdirectories?

**Comment by rhash** [comment]
Use wildcards or `--maxdepth` option:
~~~sh
rhash ./*

rhash -r --maxdepth=1 .
~~~

**Reply by i2324561w2** [reply]
> ```shell
> rhash -r ./*
> ```

That command doesn't do anything on Windows MS-DOS?! Or I'm doing something wrong? I'm using rhashV1.4.3; I also tried `.\*` but it gives the same output as `.\` listing subdirectories.
"maxdepth" works though.

**Reply by rhash** [reply]
Oh! Wildcards should be used without `-r`,  but this method prints errors on directories to stderr.
~~~bat
> rhash.exe --md5 .\*
RHash: dir1: Is a directory
d41d8cd98f00b204e9800998ecf8427e  empty-file.txt
~~~

So `rhash -r --maxdepth=1 .` is better.

**Comment by rhash** [comment, marked as answer]
Use command: `rhash -r --maxdepth=1 .`
